test-cases/longevity/longevity-large-partitions-with-network-nemesis-1h.yaml

```diff
@@ -30,7 +30,7 @@ instance_type_db: 'i4i.4xlarge'
 instance_type_loader: 'c7i.2xlarge'
 
 nemesis_class_name: 'SisyphusMonkey'
-nemesis_selector: ['!limited','!sla','!manager_operation', '!config_changes', 'disruptive']
+nemesis_selector: ['not limited','not sla','not manager_operation', 'not config_changes', 'disruptive']
```

Comment:
Multiple values are treated as an OR?

Reply:
they are treated as an AND, as before this change

Reply:
and yes, this can be written as the following:
`disruptive and not limited and not sla ...`

Reply:
and also can be:

```
disruptive and not (sla or limited or manager_operation or config_changes)
```

which is probably the most readable option

Reply:
now that I think of it, there's one more thing missing to this syntax 

ability to add the name of the class or the method

it should be quite easy addition, I think..

Reply:
> and also can be:
> 
> ```
> disruptive and not (sla or limited or manager_operation or config_changes)
> ```
> 
> which is probably the most readable option

I think we should disallow using list instead of a logical statement and we should rewrite all conditions to be like this.

Reply:
I would need a bit more refactoring

Reply:
I putting this one as draft for now

I also want to add a possibility to filter base on nemesis class or functions names, that is also something asked needed for quite some time.

Reply:
took it out or draft